include/linux/vhm/acrn_common.h

```diff
@@ -178,10 +178,10 @@ struct acrn_create_vm {
  */
 struct acrn_create_vcpu {
 	/** the virtual CPU ID for the VCPU created */
-	uint32_t vcpu_id;
+	uint16_t vcpu_id;
```

Comment:
I don't think that it is a good idea to change the data type of interface structure between two modules.
If it is changed in one module, another module will be broken before it is changed. 
This will bring the side effect on bisecting.